heroic-core/src/main/java/com/spotify/heroic/HeroicCore.java

```diff
@@ -613,7 +611,10 @@ private HeroicConfig config(LoadingComponent loading) throws Exception {
             builder = builder.merge(fragment);
         }
 
-        final ObjectMapper mapper = loading.configMapper();
+        final ObjectMapper mapper = loading.configMapper().copy();
+
+        // TODO: figure out where to put this
```

Comment:
Shall we do something about this now, or do it later?
If it's not critical, I vote for doing incremental changes and thus letting this be for now.